9c-internal/argocd/bootstrap.yaml

```diff
@@ -116,6 +116,10 @@ spec:
           nodeGroup: 9c-internal-ondemand-r7g_l_2c
         nginxGatewayFabric:
           enabled: true
+          awsLoadBalancerSubnets: public-us-east-2c-9c-internal
+          awsLoadBalancerSslCert: >-
+            arn:aws:acm:us-east-2:319679068466:certificate/e19257bd-f1fb-41a6-ae84-bbdf6b98a62f
```

Comment:
can we serve multiple root domains with single cert?

Reply:
yes, if the cert is issued for multiple domains